backend/sokdak/src/main/java/com/wooteco/sokdak/member/util/RandomNicknameGenerator.java

```diff
@@ -20,12 +24,13 @@ public class RandomNicknameGenerator {
     }
 
     public static String generate(Set<String> usedNicknames) {
-        return RANDOM_NICKNAMES.stream()
+        List<String> usableNicknames = RANDOM_NICKNAMES.stream()
                 .filter(nickname -> !usedNicknames.contains(nickname))
-                .findFirst()
-                .orElse("크루");
-    }
+                .collect(Collectors.toList());
 
-    private RandomNicknameGenerator() {
+        if (usableNicknames.size() > 0) {
+            return usableNicknames.get(RANDOM.nextInt(usableNicknames.size()));
+        }
+        return "크루";
```

Comment:
```suggestion
        if (usable nickname.isEmpty()) {
            return "크루";
        }
        return usableNicknames.get(RANDOM.nextInt(usableNicknames.size()));
```
이 메서드의 목적은 랜덤한 닉네임을 생성하는 것이니 위와 같이 변경하면 더 명확할 것 같아요!

Reply:
오 이게 더 좋네요 수정했습니다!